blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json

```diff
@@ -375,12 +411,53 @@
           }
         },
         "width": 6,
+        "xPos": 6,
+        "yPos": 16
+      },
+      {
+        "height": 4,
+        "widget": {
+          "title": "firewalls_per_project_vpc_usage",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "minAlignmentPeriod": "60s",
+                "plotType": "STACKED_BAR",
```

Comment:
Can you use the following instead (like the other charts):
"plotType": "LINE",

Reply:
y